src/data/set/lattice.lean

```diff
@@ -153,6 +153,10 @@ theorem antitone_set_of [preorder α] {p : α → β → Prop}
   (hp : ∀ b, antitone (λ a, p a b)) : antitone (λ a, {b | p a b}) :=
 λ a a' h b, hp b h
 
+/-- Quantifying over a set is antitone in the set -/
+lemma antitone_ball {P : α → Prop} : antitone (λ s : set α, ∀ x ∈ s, P x) :=
```

Comment:
I know we have some lemmas in mathlib with this `ball` (i.e., bounded forall), but I've always found it confusing at is sounds like a ball in a metric space. At least `bforall` would look less ambiguous to me.

Reply:
I agree, I will change it to `bforall`, and we can try to propagate that change also in other places.